docs/_posts/2017-06-13-react-v15.6.0.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "React v15.6.0"
+author: flarnie
+---
+
+Today we are releasing React 15.6.0. As we prepare for React 16.0, we have been fixing and cleaning up many things. This release continues to pave the way.
+
+## Improving Inputs
+
+In React 15.6.0 the `onChange` event for inputs is a little bit more reliable and handles more edge cases, including the following:
+
+* not firing when radio button is clicked but not changed ([issue 1471](https://github.com/facebook/react/issues/1471))
+* changing an input of type 'range' with the arrow keys ([issue 554](https://github.com/facebook/react/issues/554))
+* pasting text into a text area in IE11 ([issue 7211](https://github.com/facebook/react/issues/7211))
+* auto-fill in IE11 ([issue 6614](https://github.com/facebook/react/issues/6614))
+* clearing input with 'x' button or right-click 'delete' in IE11 ([issue 6822](https://github.com/facebook/react/issues/6822))
+* not firing when characters are present in the input on render in IE11 ([issue 2185](https://github.com/facebook/react/issues/2185))
+
+Thanks to [Jason Quense](https://github.com/jquense) and everyone who helped out on those issues and PRs.
+
+## Downgrading Deprecation Warnings
+
+We are also including a couple of new warnings for upcoming deprecations. These should not affect most users, and for more details see the changelog below.
+
+After the last release, we got valuable community feedback that deprecation warnings were causing noise and failing tests. **In React 15.6, we have downgraded deprecation warnings to use `console.warn` instead of `console.error`.** Our other warnings will still use `console.error` because they surface urgent issues which could lead to bugs. Unlike our other warnings, deprecation warnings can be fixed over time and won't cause problems in your app if shipped. We believe that downgrading the urgency of deprecation warnings will make your next update easier. Thanks to everyone who was involved in the discussion of this change.
+
+---
+
+## Installation
+
+We recommend using [Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/) for managing front-end dependencies. If you're new to package managers, the [Yarn documentation](https://yarnpkg.com/en/docs/getting-started) is a good place to get started.
+
+To install React with Yarn, run:
+
+```bash
+yarn add react@^15.6.0 react-dom@^15.6.0
+```
+
+To install React with npm, run:
+
+```bash
+npm install --save react@^15.6.0 react-dom@^15.6.0
+```
+
+We recommend using a bundler like [webpack](https://webpack.js.org/) or [Browserify](http://browserify.org/) so you can write modular code and bundle it together into small packages to optimize load time.
+
+Remember that by default, React runs extra checks and provides helpful warnings in development mode. When deploying your app, make sure to [use the production build](https://facebook.github.io/react/docs/optimizing-performance.html#use-the-production-build).
```

Comment:
Generally we don't leave domain in posts in case we change the domain. So the link should start with `/react/docs/...` instead.